internal/flowcontrol/writebuffer.go

```diff
@@ -99,31 +104,43 @@ func (w *ResourceSliceWriteBuffer) processQueueItem(ctx context.Context) bool {
 	ctx = logr.NewContext(ctx, logger)
 
 	w.mut.Lock()
+	insertionTime := w.insertionTime[sliceNSN]
 	updates := w.state[sliceNSN]
 	delete(w.state, sliceNSN)
+
+	// Limit the number of operations per patch request
```

Comment:
Where is the limit coming from?

Reply:
This beauty: https://github.com/kubernetes/apiserver/blob/51262c6edb897d1063d2b02183895a10f7132b8f/pkg/endpoints/handlers/patch.go#L60

I thought the request body size would be too large anyway but the integration test proves otherwise 🤷 . It works perfectly right at the limit.

Reply:
Lol, wonder whey they don't export that var...